### PR TITLE
Refactor with classes

### PR DESCRIPTION
**Pull request by Obote**
### **What does this repo do?**

- Use JavaScript classes instead of objects and arrays.
- Create class methods to add and remove books.
- Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
- Don't use any library or framework, use only plain JavaScript.
- Add CSS styles to the application to make it match this wireframe.
- You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.

### CODE REVIEW

File: JS/index.js [hunk 1]
```diff
@@ -1,99 +1,70 @@
...
   }
-  if (error.classList.contains('alert') && (document.querySelector('.alert') !== null)) {
-    setTimeout(() => document.querySelector('.alert').remove(), 2000);
+
+  removeFromCollection(data) {
+    const arr = data.getAttribute('data-value').split('-');
+    this.books = this.books.filter(
+      (item) => item.title + item.author !== arr[0] + arr[1],
+    );
+    this.saveToLocalStorage();
```
Comment (DammyShittu) [inline JS/index.js:52]: - [ ] There seems to be a bug in your app. When I add two books of the title and author and delete only one, both books are deleted from the local storage (even though the other book remains on the page), but that should not be. Only the deleted book should be removed from the local storage. You can fix this by deleting books using unique ids

  Duplicate books are added | One book is deleted, and both books are removed from local storage | Page is refreshed, the other book disappears
  :---:|:---:|:---:
  auth-1 | auth-2 | auth-3

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi team,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Use of classes

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: JS/index.js [hunk 2]
```diff
@@ -1,93 +1,102 @@
...
+    if (error.classList.contains('alert') && (document.querySelector('.alert') !== null)) {
+      setTimeout(() => document.querySelector('.alert').remove(), 2000);
+    }
   }
-}
 
-function removeBook(elem) {
-  if (elem.classList.contains('remove')) {
-    elem.parentElement.remove();
+  static removeBook(elem) {
```
Comment (danmainah) [inline JS/index.js:86]: - [ ] @Obote and @DhuruvSingh great work trying to fix the error, however, when I delete a book the changes made are not implemented in the local storage.Kindly check on it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Obote and @DhuruvSingh , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: JS/index.js (hunk 2 above)
Comment (Obote) [reply JS/index.js:86]: Hello, @danmainah I have been looking at this for a while now but seems I can`t figure out what you're asking for.

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @Obote, @DhuruvSingh  :wave:

Great work on implementing the changes requested by the previous reviewer. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:
:heavy_check_mark: Followed Linters
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Following the requirements

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**